gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/panels/GradeSummaryTablePanel.java

```diff
@@ -399,6 +391,19 @@ public boolean isVisible() {
 		});
 	}
 
+	private void addInstructorAttributeOrHide(WebMarkupContainer sakaiRubricPreviewButton, Long assignmentId, String studentId, boolean showingStudentView) {
+
+		if (!showingStudentView && (GradeSummaryTablePanel.this.getUserRole() == GbRole.INSTRUCTOR
+					|| GradeSummaryTablePanel.this.getUserRole() == GbRole.TA)) {
+			sakaiRubricPreviewButton.add(AttributeModifier.append("instructor", true));
```

Comment:
should this attribute be better named to "visibility"? as that is what is actually happening